bugsnag-plugin-android-ndk/src/main/jni/utils/jni_utils.h

```diff
@@ -0,0 +1,28 @@
+//
+// Created by Karl Stenerud on 22.02.21.
+//
+
+// Utils to deal with common JNI situations.
+//
+// These tools depend on the JNI cache. DO NOT call them unless
+// bsg_is_jni_cache_valid() returns true!
```

Comment:
As mentioned previously, these should either check whether the cache is valid or have it passed in as a parameter - it'd be easy for someone to assume that they already perform the check since they're utils functions.